acquia/acquia_cms
# ACMS-705: Added z-index on search button to the custom style applied.

**Motivation**
The search button on the search bar is clickable, however the search icon is not clickable and yields no result.  
Fixes #705

**Proposed changes**
- Gave search icon with z-index : 1
- Made the pointer to the search icon as cursor with cursor: pointer property. 

**Testing steps**
Click on search button, in the search field below, carefully click on the search icon. Notice that it is now clickable and searches for the content

**Merge requirements**
- [ *] _Bug_
